examples/esp_idf/settings/README.md

```diff
@@ -0,0 +1,124 @@
+# Golioth Settings Sample
+
+## Overview
+
+This sample application demonstrates how to use the Golioth Device
+Settings service to register for automatic updates when a setting is
+changed on the Golioth Cloud.
+
+## Requirements
+
+* Golioth credentials
+* Network connectivity
+
+## Building and Running
+
+### Command-line
+
+First, setup the environment. This step assumes you've installed esp-idf
+to `~/esp/esp-idf`. If you haven't, follow the initial steps in
+examples/esp_idf/README.md
+
+```sh
+source ~/esp/esp-idf/export.sh
+```
+You may have to set target based on the ESP32 chip you are using.
+For example, if you are using ESP32-C3, enter this:
+
+```
+idf.py set-target esp32c3
+```
+
+Next, `cd` to the lightdb/observe example where you can build/flash/monitor:
+```
+cd examples/esp_idf/lightdb/observe
+idf.py build
+idf.py flash
+idf.py monitor
+```
+## Adding credentials
+
+After building and flashing this app, you will need to add WiFi and Golioth
+credentials using the device shell.
+
+### Shell Credentials
+
+```console
+settings set wifi/ssid "YourWiFiAccessPointName"
+settings set wifi/psk "YourWiFiPassword"
+settings set golioth/psk-id "YourGoliothDevicePSK-ID"
+settings set golioth/psk "YourGoliothDevicePSK"
+```
+
+Type `reset` to restart the app with the new credentials.
+
+
+### Receiving remote settings updates
+
+This sample registers to receive settings updates made to the
+`LOOP_DELAY_S` key. To create this key, go to the [Golioth Web
+Console](https://console.golioth.io) and select `Device Settings` in the
+left sidebar. Click on the `Create` button in the upper right and enter
+your settings values in the modal window:
+
+* Key: `LOOP_DELAY_S`
+* Data Type: `Integer`
+* Value: your desired delay (in seconds) between each "hello" message
+  * this sample tests received values for a range of 1 to 300
+
+This creates a value that all devices in your fleet will receive
+(project-wide). The setting may be changed at the
+blueprint level, and per-device. The device will receive the settings
+value each time it powers up, and whenever the value is updated on the
+cloud.
+
+
+### Sample output
+
+```console
+*** Booting Zephyr OS build zephyr-v3.4.0-553-g40d224022608 ***
```

Comment:
This looks leftover from the Zephyr example?

Reply:
Yes! Thanks for catching it.